core/chains/evm/config/toml/defaults/BSC_Mainnet.toml

```diff
@@ -2,32 +2,19 @@
 # Clique offers finality within (N/2)+1 blocks where N is number of signers
 # There are 21 BSC validators so theoretically finality should occur after 21/2+1 = 11 blocks
 ChainID = '56'
-# Keeping this >> 11 because it's not expensive and gives us a safety margin
```

Comment:
Maybe this comment ought to remain, to show why we chose to go with eth's depth for BSC too?